tests/unit/test_magics.py

```diff
@@ -33,7 +33,7 @@
 try:
     from google.cloud.bigquery.magics import magics
 except ImportError:
-    magics = None
+    raise  # magics = None
```

Comment:
I'm assuming this is some temporary debugging thing that leaked into the PR.  The point here is to have a bool you can use for skipif annotations.  You'll want to do the same with ipywidgets, so you only execute the test if the optional dependency is present.

Reply:
addressed in latest PR!